Training/2.0 Libraries.md

```diff
@@ -25,14 +25,20 @@ Make a two-dimensional numpy array. First row will be: [1,2,3] and second row wi
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
 ```
-
+x = numpy.array([1,2,3],[4,5,6])
+max = x.max()
+min = x.min()
+mean = x.mean()
 ```
 
 ### Task 2b)
 Calculate the trimmed mean of the following array using ONLY Numpy and python's base functions: [67,67,65,50,69,68,67,70,69,66]. The trimmed mean is the mean of the data with the largest and the smallest values (outliers) removed.
 
 ```
-
+x = np.array([67,67,65,50,69,68,67,70,69,66])
+max = x.max()
+min = x.min()
+trim = (max-min)/2
 ```
 
```

Comment:
The trimmed mean is not calculated like that - you should remove the max and min values from the array and then calculate the average of the remaining values of the array. Give it another go, and if you need any help let either Hasitha or myself know so we can explain it.